CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# CHANGELOG for `contextual_logger`
+
+Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.5.1.pre.1] - 2019-03-10
+
+### Changed
+
+- Refactored debug, info, error... etc methods to call the base class `add(severity, message, progrname)` method since
+  ActiveSupport::Logger.broadcast reimplements that to broadcast to multiple logger instances, such as
+  Rails::Server logging to `STDOUT` + `development.log`.
+  Note that the base class `add()` does not have a `context` hash like our `add()` does.
+  We use the `**` splat to match the `context` hash up to the extra
+  `**context` argument, if present. If that argument is not present (such as with `broadcast`), Ruby will instead
+  match the `**context` up to the `progname` argument.
+
+## [0.5] - 2019-03-06
```

Comment:
Can these be made into diff links so that it's easy to jump to the change diff in the code? Example for this one would be to add to the bottom of the file
```
[0.5]: https://github.com/Invoca/contextual_logger/compare/v0.4.0...v0.5.0
```

Reply:
Yes, they could. Are you suggesting we always write those?

Reply:
Yea it's very helpful to link from the version to the code change diff.  If you follow the Changelog spec that you're linking to, link pointers are accumulated at the bottom of the file, and the version number is what links to the change diff, rather than a `diff` hyperlink adjacent to it.

Reply:
I followed the Changelog spec as far as "There is no standard format." And then I clicked on their exemplar project which led me here:
https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md

If we're going to link to diffs, might as well link to them where they're most relevant. Given the most-recent-first convention, people aren't going to think to scroll to the bottom. I sure didn't.